buddyboss/buddyboss-platform
# #3188 : Group > Private message > All the users are not displayed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3188 .

### How to test the changes in this Pull Request:

1. Go to 'Group private message'
2. Scroll down 'user list'
3. See error

**NOTE:** If users blocked then it does affect on total user count. 
- Sumit mentioned in this issue, The group has 1251 users and Send message screen appears 1221 count. 
=> The reason is bp_group_members table has duplicate user entry so, the total count appears 1251 instead of actual user count.

### Proof Screenshots or Video
https://vimeo.com/649891810/8586f36d9c


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
